pipeline/inputs/opentelemetry.md

```diff
@@ -50,13 +50,13 @@ __OTLP/GRPC__
 
 ## Getting started
 
-The OpenTelemetry plugin currently supports the following telemetry data types:
+The OpenTelemetry input plugin currently supports the following telemetry data types:
```

Comment:
The document reflects the behavior of the current product, so there's no need to say "currently". It also sets a potential expectation of different behavior in the future, which is not for documentation to do.

```suggestion
The OpenTelemetry input plugin supports the following telemetry data types:
```